### DISCUSSION TITLE
어쩔랭 v2 기존 내용 정리

### DISCUSSION THREAD

**Discussion by sangchoo1201** [assertive-lang/asserlang#52] [Ideas]
- 문자열형 변수
  - 문자열형 변수는, 다른 언어의 string같이 연속된 문자의 나열을 담는 객체이다.
  - 문자열형 변수는, 유니코드를 사용한다.
  - 문자열형 변수끼리 더하기를 하면, 두 문자열이 이어붙여진다.
  - 문자열과 다른 자료형의 연산은 불가능하다.

- 입력 방식
  - 입력은, 숫자 입력과 문자열 입력으로 구분한다.
  - 문자열 입력은, 엔터를 치기 전까지만 받는다.
  - 문자열 입력을 받는 결과를 숫자형 변수에 저장할 수는 없다.

**Comment by chul0721** [comment]
의견 정리해주셔서 감사합니다.
혹시 위 내용에 대한 자세한 내용을 알고 싶으신 분들은 디스코드 #✍v2-standard-syntax 채널에서 올라온 내용을 참고해주시기 바랍니다.
